src/api/routes/articleRouter.js

```diff
@@ -22,6 +24,19 @@ articleRouter.post('/:slug/rating',
   ratingsController.rateArticle);
 const highlightFields = ['firstIndex', 'lastIndex', 'comment', 'text'];
 
+articleRouter.patch('/:slug/comments',
+  checkValidToken,
+  checkArticle.getArticle,
+  bodyVerifier,
```

Comment:
@MCFrank16 let me break it down for you so you can get my logic
1. We check if the token is valid because the user has to login in order to comment.
2. we see if the slug exists and this tells us the article exists.
3. is he sending an empty body and if so, the user can proceed to the next process until he/she sends something we can work with.
4. Not only is the user not suppose to send an empty body but also he . has to send the required property that's ``commentBody`` then can the system proceed.
5. the comparison middleware does its magic and see if the comment updates the comment. we have or not and if it updates, he continues to the controller but else he/she stops here.